### PR TITLE
full pep8 for modular/hecke

### PR DESCRIPTION
**Pull request by fchapoton**
small cleanup for `modular/hecke` folder

### :memo: Checklist

- [x] The title is concise, informative, and self-explanatory.
- [x] The description explains in detail what this PR is about.

### CODE REVIEW

File: src/sage/modular/hecke/element.py [hunk 1]
```diff
@@ -247,14 +249,14 @@ def is_cuspidal(self):
...
         """
-        return (self in self.parent().ambient().cuspidal_submodule())
+        return self in self.parent().ambient().cuspidal_submodule()
 
-    def is_eisenstein(self):
+    def is_eisenstein(self) -> bool:
         r"""
-        Return True if this element is Eisenstein. This makes sense for both
-        modular forms and modular symbols.
+        Return True if this element is Eisenstein.
```
Comment (dcoudert) [inline src/sage/modular/hecke/element.py]: `True` -> ` ``True`` `

Also in other places

File: src/sage/modular/hecke/hecke_operator.py [hunk 2]
```diff
@@ -662,15 +664,17 @@ def _latex_(self):
             sage: ModularSymbols(Gamma0(7), 4).hecke_operator(6)._latex_()
             'T_{6}'
         """
-        return "T_{%s}"%self.__n
+        return "T_{%s}" % self.__n
 
     def _mul_(self, other):
         """
```
Comment (dcoudert) [inline src/sage/modular/hecke/hecke_operator.py]: Shouldn't it be `r"""` here ?

**Review by dcoudert — CHANGES_REQUESTED** [review]
otherwise LGTM

**Review by dcoudert — APPROVED** [review]
LGTM.

**Comment by vbraun** [comment]
thats a no

File: src/sage/modular/hecke/submodule.py [hunk 3]
```diff
@@ -446,10 +447,12 @@ def degeneracy_map(self, level, t=1):
...
-        Compute embedded dual free module if possible. In general this won't be
-        possible, e.g., if this space is not Hecke equivariant, possibly if it
-        is not cuspidal, or if the characteristic is not 0. In all these cases
-        we raise a RuntimeError exception.
+        Compute embedded dual free module if possible.
+
+        In general this will not be possible, e.g., if this space is
+        not Hecke equivariant, possibly if it is not cuspidal, or if
+        the characteristic is not 0. In all these cases we raise a
+        :classl:`RuntimeError` exception.
```
Comment (dcoudert) [inline src/sage/modular/hecke/submodule.py]: `classl` -> `class`.

**Comment by fchapoton** [comment]
oups, sorry, fixed

**Review by dcoudert — APPROVED** [review]
Sorry I didn't see this error the first time.

LGTM.

**Pull request merged**